Br cloud restore sql bindings (#15477)
pingcap/docs: tidb-cloud/backup-and-restore.md

## Limitations

> - TiDB Dedicated supports restoring user accounts and sql bindings from backups by default from TiDB v6.2.0.

Reviewer: ```suggestion
- For clusters that are v6.2.0 or later, TiDB Dedicated supports restoring user accounts and sql bindings from backups by default.
```

> - For clusters that are v6.2.0 or later, TiDB Dedicated supports restoring user accounts and sql bindings from backups by default.

Reviewer 2: ```suggestion
- For clusters of v6.2.0 or later versions, TiDB Dedicated supports restoring user accounts and SQL bindings from backups by default.
```
